com.lachhebo.Gabtag.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "com.lachhebo.Gabtag",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gabtag",
+    "finish-args": [
+        "--filesystem=home",
```

Comment:
Does your application allow to open entire directories? If so, it would be better to limit it to `xdg-music` as users can override permissions on their own if they'd like to give the app access to entire home. If it doesn't support directories, `GtkFileChooserNative` uses portal by default and it can be dropped.

Reply:
It open directories (more specifically, the user can only interact with mp3 file within those directory).